code.js

```diff
@@ -0,0 +1,83 @@
+// https://randomuser.me/api/
+
+const fetch_data = document.getElementById("details")
+
+ async function apidata(){
+    const result = await fetch("https://randomuser.me/api/")
+    const data = await result.json()
+
+    const getdata = data.results[0]
+
+    const alldata = document.createElement("div")
+    alldata.classList.add("container")
+
+    const image = document.createElement("img")
+    image.classList.add("image")
+    image.src = getdata.picture.medium
+    
+    let toggel = false
+    function imgsize(){
```

Comment:
1. there should not be any nested function declaration
2. Use more meaningful name for this function